### DISCUSSION TITLE
[BUG] Custom HTML doesn't get updated when changing pages

### DISCUSSION THREAD

**Discussion by Timtam** [rejetto/hfs#532] [Q&A]
Hello there,

I've got a setup like we discussed in #481. It works fine as far as I can see. The markdown is currently converted into HTML and shown on the main page (currently officially available on https://hoard.reaperaccessibility.com). Its displayed above the file list and that is fine. However, if I click on a file or directory and switch pages to the new subfolder page, the markdown still gets displayed and doesn't update as well. Is that intended or how could I handle this? I don't know if that is a general issue or if its just my custom HTML that is broken so I thought a new issue would be better than reopening the old one.

Thanks!

**Comment by rejetto** [comment]
i looked at your hfs and i can't see the problem you are referring to.
I also look at the code I gave you, and i see the part responsible for the reset, at every change.

**Comment by Timtam** [comment]
OK, so here is how I can reproduce it:

1. Visit https://hoard.reaperaccessibility.com. Mind the news section which appears above the file/directory listing, at least for screen readers.
2. Open a subfolder, for example Uploads.
3. The news section is still visible, at least for screen readers. I don't know if it is for sighted people too, but reading the web page from the beginning still lists the entire news section, which is the README.md file from the parent directory and thus shouldn't show up within the Uploads folder.

I hope that helps narrowing the situation down.
